.github/workflows/run-k8s-tests-on-aks.yaml

```diff
@@ -45,9 +45,18 @@ jobs:
           - host_os: cbl-mariner
             vmm: clh
             instance-type: small
+            use-policy: true
```

Comment:
USE_POLICY should be enabled for SEV, SNP, TDX and Mariner - because these are the 4 Guest images that support Policy.

AUTO_GENERATE_POLICY should be enabled for now just on Mariner. Hopefully it we will get AUTO_GENERATE_POLICY enabled for other platforms too, soon.

Reply:
Thanks Dan. I had this thought that USE_POLICY was equivalent to AUTO_GENERATE_POLICY. But I see now USE_POLICY is more meant specifically for deciding when to [run_policy_specific_tests](https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/run_kubernetes_tests.sh#L171)